src/main/java/org/elasticsearch/index/query/CommonTermsQueryBuilder.java

```diff
@@ -209,6 +281,9 @@ public void doXContent(XContentBuilder builder, Params params) throws IOExceptio
             builder.field("_name", queryName);
         }
 
+        // NO COMMIT: note that here we are always building the field disableCoord, highFreqOperator, lowFreqOperator, cutoffFrequency, 
+        // lowFreqMinimumShouldMatch, highFreqMinimumShouldMatch because of the assigned default value
```

Comment:
OK so now the behavior is not to build fields with default values. We could explain to the user that we build a more compact version of the query he has specified programmatically, skipping on default values.